File: imperative/c/time_now.c
/* Example of importing time library to get today's date and time
   https://devdocs.io/c/chrono/time
*/ 

#include <stdio.h>
#include <time.h>

int main () {
   time_t now = time(NULL);
   printf("\nHours since January 1, 1970 = %ld (epoch time)\n\n", now/3600);

   // and getting today's date/time

   printf("Right now the time is %s\n", asctime(gmtime(&now)) );
  
   return(0);
}

File: 02_imperative/python/time_now.py
# imports at the top of the file
import datetime
import time

# example of assignments
now = int(round(time.time() / 3600))   # this is in epoch time
formatted_now = datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')

print("\nHours since January 1, 1970 = {} (epoch time)\n\n".format(now));

# getting today's date/time

print("Right now the time is {}\n".format(formatted_now));
